[Bug] [JDBC Source] Exception when table have no data and partition_column is not null

### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/incubator-seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.


### What happened

Exception when table have no data and partition_column is not null

### SeaTunnel Version

2.3.1

### SeaTunnel Config

```conf
*
```


### Running Command

```shell
*
```


### Error Exception

```log
[INFO] 2023-04-13 19:05:27.408 - task execute result:2023-04-13 19:05:27,408 ERROR org.apache.seatunnel.core.starter.SeaTunnel - 

[INFO] 2023-04-13 19:05:27.408 - task execute result:

[INFO] 2023-04-13 19:05:27.408 - task execute result:===============================================================================

[INFO] 2023-04-13 19:05:27.408 - task execute result:

[INFO] 2023-04-13 19:05:27.408 - task execute result:

[INFO] 2023-04-13 19:05:27.409 - task execute result:2023-04-13 19:05:27,408 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Fatal Error, 

[INFO] 2023-04-13 19:05:27.409 - task execute result:

[INFO] 2023-04-13 19:05:27.409 - task execute result:2023-04-13 19:05:27,408 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Please submit bug report in https://github.com/apache/incubator-seatunnel/issues

[INFO] 2023-04-13 19:05:27.409 - task execute result:

[INFO] 2023-04-13 19:05:27.410 - task execute result:2023-04-13 19:05:27,408 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Reason:SeaTunnel job executed failed 

[INFO] 2023-04-13 19:05:27.410 - task execute result:

[INFO] 2023-04-13 19:05:27.410 - task execute result:2023-04-13 19:05:27,409 ERROR org.apache.seatunnel.core.starter.SeaTunnel - Exception StackTrace:org.apache.seatunnel.core.starter.exception.CommandExecuteException: SeaTunnel job executed failed

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:170)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.core.starter.SeaTunnel.run(SeaTunnel.java:40)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.core.starter.seatunnel.SeaTunnelClient.main(SeaTunnelClient.java:34)

[INFO] 2023-04-13 19:05:27.410 - task execute result:Caused by: org.apache.seatunnel.api.common.PrepareFailException: ErrorCode:[API-01], ErrorDescription:[Configuration item validate failed] - PluginName: jdbc, PluginType: source, Message: java.lang.NumberFormatException: null

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.connectors.seatunnel.jdbc.source.JdbcSource.prepare(JdbcSource.java:96)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.core.parse.ConnectorInstanceLoader.loadSourceInstance(ConnectorInstanceLoader.java:64)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.core.parse.JobConfigParser.sampleAnalyze(JobConfigParser.java:371)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.core.parse.JobConfigParser.parse(JobConfigParser.java:132)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.core.parse.MultipleTableJobConfigParser.parse(MultipleTableJobConfigParser.java:114)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.client.job.JobExecutionEnvironment.getLogicalDag(JobExecutionEnvironment.java:155)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.engine.client.job.JobExecutionEnvironment.execute(JobExecutionEnvironment.java:147)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	at org.apache.seatunnel.core.starter.seatunnel.command.ClientExecuteCommand.execute(ClientExecuteCommand.java:129)

[INFO] 2023-04-13 19:05:27.410 - task execute result:	... 2 more

[INFO] 2023-04-13 19:05:27.410 - task execute result: 

[INFO] 2023-04-13 19:05:27.410 - task execute result:2023-04-13 19:05:27,409 ERROR org.apache.seatunnel.core.starter.SeaTunnel - 

[INFO] 2023-04-13 19:05:27.410 - task execute
```


### Flink or Spark Version

_No response_

### Java or Scala Version

_No response_

### Screenshots

_No response_

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceFactory.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceFactory.java
@@ -155,11 +155,11 @@ public class JdbcSourceFactory implements TableSourceFactory {
                 max =
                         config.getPartitionUpperBound().isPresent()
                                 ? config.getPartitionUpperBound().get()
-                                : Long.parseLong(rs.getString(1));
+                                : rs.getLong(1);
                 min =
                         config.getPartitionLowerBound().isPresent()
                                 ? config.getPartitionLowerBound().get()
-                                : Long.parseLong(rs.getString(2));
+                                : rs.getLong(2);
             }
         } catch (SQLException e) {
             throw new PrepareFailException("jdbc", PluginType.SOURCE, e.toString());

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceSplitEnumerator.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceSplitEnumerator.java
@@ -108,7 +108,8 @@ public class JdbcSourceSplitEnumerator
                             : enumeratorContext.currentParallelism();
             JdbcNumericBetweenParametersProvider jdbcNumericBetweenParametersProvider =
                     new JdbcNumericBetweenParametersProvider(
-                                    partitionParameter.getMinValue(), partitionParameter.getMaxValue())
+                                    partitionParameter.getMinValue(),
+                                    partitionParameter.getMaxValue())
                             .ofBatchNum(partitionNumber);
             Serializable[][] parameterValues =
                     jdbcNumericBetweenParametersProvider.getParameterValues();

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceSplitEnumerator.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/JdbcSourceSplitEnumerator.java
@@ -108,7 +108,7 @@ public class JdbcSourceSplitEnumerator
                             : enumeratorContext.currentParallelism();
             JdbcNumericBetweenParametersProvider jdbcNumericBetweenParametersProvider =
                     new JdbcNumericBetweenParametersProvider(
-                                    partitionParameter.minValue, partitionParameter.maxValue)
+                                    partitionParameter.getMinValue(), partitionParameter.getMaxValue())
                             .ofBatchNum(partitionNumber);
             Serializable[][] parameterValues =
                     jdbcNumericBetweenParametersProvider.getParameterValues();

--- a/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/PartitionParameter.java
+++ b/seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/source/PartitionParameter.java
@@ -27,7 +27,7 @@ import java.io.Serializable;
 public class PartitionParameter implements Serializable {
 
     String partitionColumnName;
-    Long minValue;
-    Long maxValue;
+    long minValue;
+    long maxValue;
     Integer partitionNumber;
 }

